code-challenges.test.js

```diff
@@ -13,106 +13,39 @@
 // --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
 
 // a) Create a test with an expect statement using the variable provided.
-// Psuedo code: Im going to start my test first im going to describe here im goign to use sentenceCap, after that i use
-// "it" here im going to write wgat i want in return and i want a new array with a sentencew about each person from my previws array with their names capitallized.
-//In my expect im going to wtite the outpit that i want.
-describe("sentenceCap", () =>{
-it("returns an array with a sentence about each person with their name capitalized", () => {
-  let people = [
-    { name: "ford prefect", occupation: "a hitchhiker" },
-    { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-    { name: "arthur dent", occupation: "a radio employee" }
-  ]
-  expect(sentenceCap(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
-  })
-})
-//RED now i have to make this pass
-// b) Create the function that makes the test pass.
-//First im going to declare my vatiable sentenceCap and create my function
-//I think im going to use map so i can go around my array the same size.
-//I think i have to use .split because i want to separete my array and use toUpperCase so i can have my names capitalized.
-//im going to use .join and for my return im going to use string interpolation so i can return all my output.
-//Im going to split my name so i can make my index 0 . toUpperCase and after that im going to use .join.
-
-
-const sentenceCap =(array, value) => {
-  return array.map(value => {
-    let newCase = value.name.split(" ").map(value => value[0].toUpperCase() + value.slice(1)).join(" ")
-     return `${newCase} is ${value.occupation}`
-  })
-}
-//So my test fail
-// I had trouble with this one i got stuck i ask for help but i still cant make it work :(
```

Comment:
Your logic looks good. I would be curious to know your error message. I have a feeling it has to do with line 38. `const sentenceCap =(array)` should be sufficient.